tools/distrib/python/grpcio_tools/BUILD.bazel

```diff
@@ -37,12 +38,20 @@ pyx_library(
     deps = [":protoc_lib"],
 )
 
+internal_copied_filegroup(
+    name = "well_known_protos",
+    srcs = ["@com_google_protobuf//:well_known_protos"],
+    dest = "grpc_tools/_proto/",
+    strip_prefix = "src/",
+)
```

Comment:
TIL that Bazel built `grpcio_tools` doesn't support well-known protos before.

Reply:
It hasn't mattered in the past because `grpcio-tools` is only a library under Bazel, whereas the wheel is mostly used as a  CLI utility. To be honest, I thought that the well-known types were added into the descriptor DB and linked in without us vendoring a copy of the protos, but it turns out my manual test was affected by a cached copy of the `_proto/` directory on my filesystem.